l1-contracts/README.md

```diff
@@ -31,6 +32,16 @@ We use `forge fmt` to format. But follow a few general guidelines beyond the sta
   - Do `function transfer(address _to, uint256 _amount);`
 - use `_` prefix for `internal` and `private` functions.
 
+## Gas snapshots and CI
+
+CI will run `forge snapshot --check`. This means that as you develop, you should run `./bootstrap.sh snapshot --diff` to make sure you understand the gas cost of your changes.
+
+Note: the only reason `forge snapshot` is wrapped in the bootstrap script is to standardize the output, or any future filtering.
+
+When your PR is ready for review, run `./bootstrap.sh snapshot` to update the snapshot, then `./bootstrap.sh test` to make sure you're good.
+
+You can also run `./bootstrap.sh gas_report` to get a gas report for the current state.
```

Comment:
Think it would be fine to mention the `FORGE_GAS_REPORT` here. Also we should just have a general note that a bunch of the blob logic is skipped in the gas reports so they are not exact